Add enzymology domain page (#1614)
elixir-europe/rdmkit: pages/your_domain/enzymology_and_biocatalysis.md

### Considerations
Prerequisite for the reproducibility of enzyme activity datasets is the reporting in a complete way, without omissions and without the lack of essential information that allows other researchers to corroborate, interpret and reuse the data. Therefore, the major questions to fulfill these aspects include:
* Which data are required to provide complete data sets?
* What is the minimal data accepted to be considered complete?
* What is the minimal data set required to make the data useful for studies of metabolic pathways?
* Which metadata describe both the materials and methods data and results data most appropriately?
* From where can I obtain the metadata?
* What should be the best way to define the metabolic energy of reactions, substrates and metabolites?
* How can I address thermodynamic issues in my experiments?
* How can I ensure that my datasets are reproducible?

### Solutions
The Standards for Reporting Enzymology Data (STRENDA) Commission, a community driven initiative has developed reporting standards that are unique for this domain and include

> * [STRENDA Guidelines List Level 1a - Data required for a complete description of an experiment](https://www.beilstein-institut.de/en/projects/strenda/guidelines/#Level1A)

Reviewer: Wait, it looks like it exists already: {% tool "standards-for-reporting-enzyme-data"%}

> * [STRENDA Guidelines List Level 1a - Data required for a complete description of an experiment](https://www.beilstein-institut.de/en/projects/strenda/guidelines/#Level1A)

Author: @wittigue Also this one can still be replaced with the proposed snippet

> * [STRENDA Guidelines List Level 1a - Data required for a complete description of an experiment](https://www.beilstein-institut.de/en/projects/strenda/guidelines/#Level1A)

Reviewer 2: The idea here was to distinguish between the two levels Level1A and Level1B which can not be separated by using the tool snippet.

> * [STRENDA Guidelines List Level 1a - Data required for a complete description of an experiment](https://www.beilstein-institut.de/en/projects/strenda/guidelines/#Level1A)

Reviewer: I see, but STRENDA should appear at least once as a tool on the page, so that it goes in the table on this page and we can link it to FAIRsharing (https://fairsharing.org/FAIRsharing.8ntfwm).

> * [STRENDA Guidelines List Level 1a - Data required for a complete description of an experiment](https://www.beilstein-institut.de/en/projects/strenda/guidelines/#Level1A)

Reviewer 2: Done

The STRENDA Guidelines which initially propose reporting fundamental enzymology data has been extended by additional parameters for the description of biocatalysis experiments such as for reactors and vessels, mixing conditions and the formulation of the biocatalyst which can be studied in a solved or immobilized form. Not only parameters have been defined but also the corresponding metadata which assist researchers in describing both the biocatalyst and the reaction conditions in more detail.

The nomenclature of an enzyme should follow the systematic classification and numbering recommended by the [Enzyme Commission (EC)](https://iubmb.qmul.ac.uk/enzyme/rules.html) of the International Union of Biochemistry and Molecular Biology (IUBMB). The EC number classifies enzymes based on the catalyzing reaction. If a new enzyme is discovered or the classification changes an EC number can be proposed for reviewing at [https://www.enzyme-database.org/newform.php](https://www.enzyme-database.org/newform.php).

## Data structure and exchange
 
### Description

> When sampling and collecting data for the purpose of deposition, sharing and exchanging, the data needs to be structured in a way that both the sender and the recipient of the data are enabled to directly integrate these into their own workflow, i.e. data exchange formats, repositories, databases etc. When structuring data, usually ontologies and metadata catalogs provide a valuable means for the integration of controlled vocabularies, ontologies and additional information that enriches the experimental data. Structured data is required that follows community-based principles to increase the findability of the data in the web.

Reviewer: ```suggestion
When sampling and collecting data for deposition, sharing, and exchanging, the data needs to be structured in a way that both the sender and the recipient of the data are enabled to directly integrate these into their workflow. When structuring data, usually ontologies and metadata catalogs provide a valuable means for the integration of controlled vocabularies, ontologies and additional information that enriches the experimental data. Structured data is required that follows community-based principles to increase the findability of the data on the web.
```

Reviewer: The "i.e." bit was used in a confusing way in the sentence. I removed it for simplicity, but feel free to bring it back if rephrasing the sentence.

### Considerations

> Before data can be sampled in a structured way, frameworks and tools are required to assist the researcher in compiling complete and high-quality datasets. Therefore, the major questions that address the requirement of standardized and structured data include:

Reviewer: ```suggestion
Before data can be sampled in a structured way, frameworks and tools are required to assist the researcher in compiling complete and high-quality datasets. Therefore, the major questions that address the requirement of standardised and structured data include:
```

* How to assist with the implementation of repositories, databases and electronic lab notebooks?
* How to structure the data?
* Which ontologies to use?
* How to include ontologies in the definition of data fields?
* How to define metadata so as to enrich experimental data optimally?
* How to increase the findability of datasets?
* How to enhance the utility of data for users?

### Solutions

> {% tool "enzymeml" %} is a data exchange format enabling the transfer of enzyme function data between instruments, electronic lab notebooks, modelling tools, and databases. It includes experimental raw data as well as metadata compliant with the STRENDA guidelines and {% tool "systems-biology-markup-language" %}.

Reviewer: ```suggestion
{% tool "enzymeml" %} is a data exchange format enabling the transfer of enzyme function data between instruments, electronic lab notebooks, modelling tools, and databases. It includes experimental raw data as well as metadata compliant with the {% tool "standards-for-reporting-enzyme-data" %} and {% tool "systems-biology-markup-language" %}.
```

* Enzyme-related repositories which enable the direct submission of enzyme function data and experimental conditions are {% tool "strenda-db" %} and {% tool "sabiork" %} via the data exchange formats {% tool "enzymeml" %} and {% tool "systems-biology-markup-language" %}. 
* {% tool "strenda-db" %} is a storage and search platform for authors who are preparing a manuscript containing functional enzymology data. Data sets entered in {% tool "strenda-db" %} are automatically checked for compliance with the {% tool "standards-for-reporting-enzyme-data" %} and prepared for publication after the journal’s reviewing process together with the corresponding paper. The direct deposition of experimental data in {% tool "strenda-db" %} by the authors not only ensures the completeness of information but also simplifies the integration of the data into other enzyme databases.
* Information about the effects of chemical compounds on enzyme protein targets can be uploaded to the {% tool "chembl" %} database.   

> * Most of the databases containing enzymology data are based on information manually extracted from literature. The structured format of the literature data in such databases allows the export and reuse of enzyme data (e.g. kinetic parameters) as well as the automatic integration in processing tools for modelling or visualization. Manually curated databases containing enzyme function data are: {% tool "uniprot" %}, {% tool "brenda" %}, {% tool "sabiork" %}, {% tool "m-csa" %}, {% tool "ezcatdb" %}, {% tool "metacyc" %}. An overview about general and more specialized enzyme databases is given in [Enzyme Databases in the Era of Omics and Artificial Intelligence](https://doi.org/10.3390/ijms242316918).

Reviewer: ```suggestion
* Most of the databases containing enzymology data are based on information manually extracted from literature. The structured format of the literature data in such databases allows the export and reuse of enzyme data (e.g. kinetic parameters) as well as the automatic integration in processing tools for modelling or visualisation. Manually curated databases containing enzyme function data are: {% tool "uniprot" %}, {% tool "brenda" %}, {% tool "sabiork" %}, {% tool "m-csa" %}, {% tool "ezcatdb" %}, {% tool "metacyc" %}. An overview of general and more specialised enzyme databases is given in [Enzyme Databases in the Era of Omics and Artificial Intelligence](https://doi.org/10.3390/ijms242316918).
```

> * Most of the databases containing enzymology data are based on information manually extracted from literature. The structured format of the literature data in such databases allows the export and reuse of enzyme data (e.g. kinetic parameters) as well as the automatic integration in processing tools for modelling or visualization. Manually curated databases containing enzyme function data are: {% tool "uniprot" %}, {% tool "brenda" %}, {% tool "sabiork" %}, {% tool "m-csa" %}, {% tool "ezcatdb" %}, {% tool "metacyc" %}. An overview about general and more specialized enzyme databases is given in [Enzyme Databases in the Era of Omics and Artificial Intelligence](https://doi.org/10.3390/ijms242316918).

Reviewer: > [Enzyme Databases in the Era of Omics and Artificial Intelligence](https://doi.org/10.3390/ijms242316918).

Should probably be part of a proper Bibliography on this page.

Author: @wittigue You need help with this? Seems like an easy oen to still do before merging

> * Besides literature based information there are databases e.g. {% tool "gotenzymes" %} containing predicted kinetic parameters for enzymes or {% tool "topenzyme" %} containing predicted enzyme structure models. 

Reviewer: ```suggestion
* Besides literature-based information, there are databases such as {% tool "gotenzymes" %} containing predicted kinetic parameters for enzymes or {% tool "topenzyme" %} containing predicted enzyme structure models. 
```

> * Further repositories which are neither domain specific nor store data in a structured way are % tool "zenodo" %}, % tool "figshare" %} and % tool "dataverse" %}. These repositories are often suggested by journals and funding agencies that may be unaware of awareness of the structured repositories mentioned above.

Reviewer: ```suggestion
* Further repositories which are neither domain specific nor store data in a structured way are {% tool "zenodo" %}, {% tool "figshare" %} and {% tool "dataverse" %}. These repositories are often suggested by journals and funding agencies that may be unaware of awareness of the structured repositories mentioned above.
```

## Data processing
 
### Description

> The primary data that emerge directly from experimental analysis are mostly extensive tables of observations (e.g. measured concentrations) as a function of time or of variations in conditions such as substrate concentrations. The functional data of enzyme kinetics and biocatalysis is a much reduced set, typically comprising kinetic and thermodynamic parameters such as equilibrium dissociation constants and Michaelis-Menten and catalytic constants. These parameters completely pin down mathematical models for the biocatalysts. These models then enable the calculation of the rate of the reaction catalyzed by the enzyme for a wide range of concentrations of substrates, products, modifiers, pH values and temperatures. A frequent example of such a model is the reversible Michaelis-Menten equation. The appropriateness of this model for the description of the biocatalyst function is often uncertain or an approximation. It is important that the model used is validated for the biocatalyst under consideration.

Reviewer: ```suggestion
The primary data that emerge directly from experimental analysis are mostly extensive tables of observations (e.g. measured concentrations) as a function of time or of variations in conditions such as substrate concentrations. The functional data of enzyme kinetics and biocatalysis is a much-reduced set, typically comprising kinetic and thermodynamic parameters such as equilibrium dissociation constants and Michaelis-Menten and catalytic constants. These parameters completely pin down mathematical models for the biocatalysts. These models then enable the calculation of the rate of the reaction catalysed by the enzyme for a wide range of concentrations of substrates, products, modifiers, pH values and temperatures. A frequent example of such a model is the reversible Michaelis-Menten equation. The appropriateness of this model for the description of the biocatalyst function is often uncertain or an approximation. It is important that the model used is validated for the biocatalyst under consideration.
```

* How to infer the kinetic and thermodynamic parameters from the primary data?
* How to assess the accuracy and reliability of the kinetic and thermodynamic parameters?
* How and to what extent are experimental tests required for the validity of these kinetic models for the particular biocatalysts?

### Solutions
* Many examples of kinetic models for enzymes are available in standard enzyme kinetics textbooks. Moreover, simulation tools such as {% tool "copasi" %} and {% tool "jws-online" %} provide a predefined list of reference kinetic models from which the user can choose when entering a new enzyme reaction.
* {% tool "enzymeml" %} provides a standard format and data model for capturing the raw data, the metadata of the kinetic experiment, the kinetic model used for fitting and the final kinetic and thermodynamic parameters.
* The {% tool "pyenzyme" %} library facilitates the processing of data by providing a programmatic interface to {% tool "enzymeml" %}. {% tool "pyenzyme" %} also interfaces with computational systems biology tools such as {% tool "pysces" %} and {% tool "copasi" %} to assist with numerically solving the kinetic models for parameter estimation.

> * Fitting statistics such as the Akaike Information Criterion (AIC) or Bayesian Information Criterion (BIC), which are commonly reported by fitting libraries, can be used to select the most appropriate model from different alternatives.

Reviewer: Maybe add some links/references

Author: @wittigue One last to do :)

Reviewer 2: "Akaike Information Criterion (AIC)" and "Bayesian Information Criterion (BIC)" are general statistical methods. If I would add a link it would be maybe a Wikipedia page. I don't think this is useful. What do you think? @bedroesb @bianchini88

Reviewer: I would rather have a link.
